src/FileList/FileExtInfoLoader.cc

```diff
@@ -8,8 +8,10 @@
 
 #include "FileExtInfoLoader.h"
 
-#include <fmt/format.h>
-#include <fmt/ostream.h> // not needed here but *must* include before miriad.h
+#include <spdlog/spdlog.h>
+
+#include <spdlog/fmt/fmt.h>
+#include <spdlog/fmt/ostr.h> // not needed here but *must* include before miriad.h
```

Comment:
Should we not be using the original fmtlib headers from `spdlog/fmt/bundled/format.h` and `spdlog/fmt/bundled/ostream.h`?